server/service/src/rnr_form/tests/insert.rs

```diff
@@ -252,7 +262,7 @@ mod query {
         RnRFormRowRepository::new(&context.connection)
             .upsert_one(&RnRFormRow {
                 status: RnRFormStatus::Finalised,
-                ..mock_rnr_form_a()
+                ..mock_rnr_form_b()
```

Comment:
Edits to this file were a bit of a side quest changing some inputs after adding more mock data